add quick installation & welcome sections to readme (#135)
simplex-chat/simplex-chat: README.md

See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.

### Quick installation

```sh
curl -o- https://raw.githubusercontent.com/simplex-chat/simplex-chat/master/install.sh | bash
```

![simplex-chat](./images/connection.gif)

### Welcome word

> **We are working hard to create the most secure and user privacy focused chat in the world** - open source, decentralized, and without the need for global identities of any kind. If you believe in the project and would like to support it, you can do so in following ways:

Reviewer: ```suggestion
**We are building the most private and secure chat in the world** - open-source, decentralized, and without global identities of any kind. If you would like to support it, you can do so in the following ways:
```

> - 🌟 **Give the project a star on GitHub** - it's a great feedback we're on the right track and helps us raise the visibility of the project.

Reviewer: ```suggestion
- 🌟 **Star it on GitHub** - it's a great feedback you like it and helps us raise the visibility of the project.
```

> - **Install the chat and try it out** - if you find a bug you may help us out by [raising an issue](https://github.com/simplex-chat/simplex-chat/issues).

Reviewer: ```suggestion
- **Install the chat and try using it** - if you spot a bug it would help a lot if you could [raise an issue](https://github.com/simplex-chat/simplex-chat/issues).
```

> - **Spread the word** - terminal chat is a [prototype](#disclaimer) and our playground for testing out the protocol, but if you like it you can invite your friends for some fun chatting inside your terminal. For example, we've been using it right inside our IDEs 👨‍💻

Reviewer: ```suggestion
- **Spread the word** - terminal chat is an [early-stage product](#disclaimer) while we stabilize the protocol - you can invite your friends for some fun chatting inside your terminal. We're using it right inside our IDEs as we are coding it 👨‍💻
```

> - **Make a donation** via [opencollective](https://opencollective.com/simplex-chat) - no matter the size, we appreciate every donation!

Reviewer: ```suggestion
- **Make a donation** via [opencollective](https://opencollective.com/simplex-chat) - the amount doesn't matter, we appreciate any donation!
```
